sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ui/setup/SetupAccountActivity.kt

```diff
@@ -176,6 +183,10 @@ class SetupAccountActivity : DuckDuckGoActivity(), SetupFlowListener {
         viewModel.onRecoverAccount()
     }
 
+    private fun launchSyncGetOnOtherPlatforms() {
+        startActivity(SyncGetOnOtherPlatformsActivity.intent(this))
```

Comment:
Why not use GlobalActivityStarter?

Reply:
👍. This will be done in the upper branch: https://github.com/duckduckgo/Android/pull/4921